core/runtime/src/main/java/io/quarkus/runtime/LocalesBuildTimeConfig.java

```diff
@@ -46,6 +46,7 @@ public class LocalesBuildTimeConfig {
      * Native-image build uses this property to derive {@code user.language} and {@code user.country} for the application's
      * runtime.
      */
-    @ConfigItem(defaultValue = DEFAULT_LANGUAGE + "-" + DEFAULT_COUNTRY, defaultValueDocumentation = "Build system locale")
+    @ConfigItem(defaultValue = DEFAULT_LANGUAGE + "-"
+            + DEFAULT_COUNTRY, defaultValueDocumentation = "Build system's default locale")
```

Comment:
Meaning that's not the locale currently set on the build system but the default one of the build system?

If I do, `export LANG=fr_FR.UTF8` in my term on a `en_US.UTF-8` box, it won't affect that?

Reply:
I dunno, I would expect `LANG` to be taken into account when doing this and in this case the previous doc looked better to me.

Reply:
> I dunno, I would expect `LANG` to be taken into account when doing this and in this case the previous doc looked better to me.

Indeed `user.language` and `user.country` are being initialized based on `LC_ALL` (`LANG` seems to be ignored :/).